cmd/teleirc.go

```diff
@@ -13,14 +13,11 @@ import (
 	tg "github.com/ritlug/teleirc/internal/handlers/telegram"
 )
 
-const (
-	version = "v2.0.1"
-)
-
 var (
 	flagPath    = flag.String("conf", ".env", "config file")
 	flagDebug   = flag.Bool("debug", false, "disable debugging")
 	flagVersion = flag.Bool("version", false, "displays current version of TeleIRC")
+	version     string
```

Comment:
Out of curiosity, why declare the `version` as a `string` here? Where is this referenced, or where does it get called?

Reply:
It's injected at compile time by the shell script, that's why it's such a hack haha.